tests/e2e/Services/Databases/DatabasesCustomServerTest.php

```diff
@@ -287,15 +331,56 @@ public function testListCollections()
         $this->assertEquals($test1['body']['$id'], $collections['body']['collections'][0]['$id']);
         $this->assertEquals($test2['body']['$id'], $collections['body']['collections'][1]['$id']);
 
+        $base = array_reverse($collections['body']['collections']);
+
+        $collections = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'queries' => [ 'limit(1)' ]
+        ]);
+
+        $this->assertEquals(200, $collections['headers']['status-code']);
+        $this->assertCount(1, $collections['body']['collections']);
+
+        $collections = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'queries' => [ 'offset(1)' ]
+        ]);
+
+        $this->assertEquals(200, $collections['headers']['status-code']);
+        $this->assertCount(1, $collections['body']['collections']);
+
+        $collections = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'queries' => [ 'equal("enabled", true)' ]
+        ]);
+
+        $this->assertEquals(200, $collections['headers']['status-code']);
+        $this->assertCount(2, $collections['body']['collections']);
+
+        $collections = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'queries' => [ 'equal("enabled", false)' ]
+        ]);
+
+        $this->assertEquals(200, $collections['headers']['status-code']);
+        $this->assertCount(0, $collections['body']['collections']);
+
         /**
          * Test for Order
          */
-        $base = array_reverse($collections['body']['collections']);
         $collections = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections', array_merge([
             'content-type' => 'application/json',
             'x-appwrite-project' => $this->getProject()['$id'],
         ], $this->getHeaders()), [
-            'orderType' => 'DESC'
+            'queries' => [ 'orderDesc("$id")' ],
```

Comment:
To make this match the previous behavior, I think this should be `orderDesc("")`